docs/guides/math-blocks.mdx

```diff
@@ -86,3 +86,21 @@ module.exports = withMDX({
   pageExtensions: ['js', 'jsx', 'md', 'mdx']
 })
 ```
+
+If you're using MDX with Gatsby.js, you can use `gatsby-plugin-mdx` with the following configuration:
+```
+//gatsby-config.js
+const plugins = [
+{
+    resolve: 'gatsby-plugin-mdx',
```

Comment:
the indentation looks off here, running [`yarn format`](https://github.com/mdx-js/mdx/blob/fad9822384752999673575322f6422a04dd82a9c/package.json#L24) may resolve this.

Reply:
Fixed!